### ISSUE TITLE
Develop a new spectral search module

### ISSUE THREAD

**Issue by tsufz** [MassBank/MassBank-web#113]
Spectral search was dropped during deprecation of Java applets. @meowcat recommended to enable a new spectral search in the further development of MassBank (#111).

**Comment by schymane** [comment]
It would be nice to have the ability to access this through both copy/paste of peak lists and e.g. a “landing page” style URL. To discuss is also whether to accept batch uploads (several peak lists at once)?

**Comment by Treutler** [comment]
Implemented in quicksearch/peak search.

**Comment by tsufz** [comment]
No, this is not implemented in quicksearch/peak search. The old spectral search allowed an upload of a file containing spectral information.

**Comment by Treutler** [comment]
The match of an input spectrum against MassBank is implemented in quicksearch/peak search. Please specify further features, i.e. what kind of file should be uploaded and which interactive features are needed.
